ILSpy.ReadyToRun/ReadyToRunLanguage.cs

```diff
@@ -149,7 +152,27 @@ private void Disassemble(ITextOutput output, ReadyToRunReader reader, ReadyToRun
 				for (int i = 0; i < missingBytes; i++)
 					output.Write("  ");
 				output.Write(" ");
-				output.WriteLine(tempOutput.ToStringAndReset());
+				output.Write(tempOutput.ToStringAndReset());
+				int importCellAddress = (int)instr.IPRelativeMemoryAddress;
+				if (instr.IsCallNearIndirect && reader.ImportCellNames.ContainsKey(importCellAddress)) {
+					output.Write(" ;");
+					ReadyToRunSignature signature = reader.ImportSignatures[(int)instr.IPRelativeMemoryAddress];
+					string formattedSignatureString = reader.ImportCellNames[importCellAddress];
+					MethodDefEntrySignature methodDefSignature = signature as MethodDefEntrySignature;
```

Comment:
I think this should be converted into a type-switch on `signature` to improve readability.